ts/Base/MaterialComponent.ts

```diff
@@ -76,9 +76,11 @@ export abstract class MaterialComponent<
     }
 
     const element = this.materialDom(componentProps);
-    element.attributes = element.attributes || {};
+    // @ts-ignore
+    element.props = element.props || {};
```

Comment:
This breaks compatibility with preact 8.X.

You could probably extract this into a separate function (like `getProps()`)

Reply:
preact 8.X has no `props` right?